### ISSUE TITLE
fisherman bug

### ISSUE THREAD

**Issue by xBrunex** [esx-framework/esx_jobs#130]
Hello guys,

Can someone help me ?
In the fisherman job when i try to spawn the boat nothing happens.

How can i fix it ?

Thanks

**Comment by AdventurePandah** [comment]
You can only fish if you leave the vehicle. Just press F and then press E

**Comment by xBrunex** [comment]
yes mate i know it, but i want to fix it so that when im driving it doesn't show the message "press E to fish".

Do you know how can i fix it ?

**Comment by izio38** [comment]
This should be closed. Since it already has been asnwered in another issue.

**Comment by FBFezz** [comment]
> yes mate i know it, but i want to fix it so that when im driving it doesn't show the message "press E to fish".
> 
> Do you know how can i fix it ?

Where the notification is just add a check for if player is in a vehicle then  don't show it
